h2o-docs/src/product/grid-search.rst

```diff
@@ -85,6 +89,169 @@ Grid Search in R and Python
         {'strategy': "RandomDiscrete", 'stopping_tolerance': 0.001, 'stopping_rounds': 10}
         {'strategy': "RandomDiscrete", 'stopping_metric': "misclassification", 'stopping_tolerance': 0.0005, 'stopping_rounds': 5}
 
+Saving and Loading a Grid Search
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+H2O supports saving and loading grids even after a cluster wipe or complete cluster restart. The ``save_grid`` function will export a grid and its models into a given folder while the ``load_grid`` function loads a previously saved grid and all its models from the given folder.
+
+There are two modes to save a grid (in both R and Python):
+
+- Use auto-checkpointing and supply the ``export_checkpoitns_dir`` parameter
+- Call the new function ``h2o.save_grid`` for manual export
```

Comment:
Avoid "new" because who knows when we'll finally take that out.

- Call the ``h2o.save_grid`` function for manual export.